### ISSUE TITLE
[FEATURE] Criticals that maybe bypass all server

### ISSUE THREAD

**Issue by Fintoki** [CCBlueX/LiquidBounce#3290]
### LiquidBounce Branch

Legacy

### Describe your feature request.

This may be strange, but I asked for this suggestion to be added, namely a critical timer that can bypass intave and maybe all anti-cheats. it may sound strange but this works when we jump and immediately touch the ground the time needed is very short but by using a timer with a timer below 0.4 - 0.2 with the right height the benefits you get also give 2 critical damage which normally can only hit 1 critical I've tried it but I'm quite tired of using the timer while in the air and hitting the enemy. If it's not right, the enemy could do a combo or make you bounce very far.

**Comment by EclipsesDev** [comment]
I actually tried making this critical, but never went with it.. I'll probably look into this

**Comment by Fintoki** [comment]
> I actually tried making this critical, but never went with it.. I'll probably look into this

alr good luck

**Comment by 1zun4** [comment]
Yes I thought about this as well. Might have a high "click count" per tick, but some anti-cheats go by time.

**Comment by Husky272** [comment]
https://github.com/xia-mc/Raven-XD/blob/master/src/main/java/keystrokesmod/module/impl/combat/criticals/TimerCriticals.java

**Comment by 1zun4** [comment]
https://github.com/CCBlueX/LiquidBounce/pull/5446
